src/js/apps/patients/schedule/schedule_app.js

```diff
@@ -87,6 +87,13 @@ export default App.extend({
     this.showSearchView();
     this.toggleBulkSelect();
   },
+  onChangeFilters() {
+    this.getState().clearSearchQuery();
```

Comment:
I wonder if this cleanup should be in an `onStop`. could even be wrapped in an `isRestarting` I think...

Reply:
That's a better way to do it. Updated to that in my last push.